tests/system_tests/test_core/test_offline_sync.py

```diff
@@ -45,7 +46,7 @@ def test_beta_sync(wandb_init, runner, mark_synced):
         assert not os.path.exists(f"{run.settings.sync_file}.synced")
 
 
-def test_beta_sync_two_runs(user, test_settings, runner):
+def test_beta_sync_two_runs(test_settings, runner):
```

Comment:
wait, i think you need the user here?

Reply:
Looks like this test is actually being skipped. (see [this run](https://app.circleci.com/pipelines/github/wandb/wandb/40860/workflows/3ad1d09e-fc4c-44fc-b47d-f2fc2824689f/jobs/1296878/parallel-runs/3/steps/3-109)
from the latest mainline build)

running locally and Im seeing it being skipped due to 
```
SKIPPED (could not import 'wandb_core': No module named 'wandb_core')
```

It might need a user, so Ill add it back but it needs some investigation